### ISSUE TITLE
Possible error in AC-3-exp.enc

### ISSUE THREAD

**Issue by MajusCorp** [tphinney/font-tools#4]
Thomas:
One of the unencoded entries in AC-3-exp.enc actually has a Unicode value. uni0438_uni0300 == uni045D, iigrave-cy. The only noticeable difference is the naming. uni0438_uni0300 translates as ii_gravecomb-cy using OS X Services. The same situation exists for the cap version, uni0418_uni0300 == uni040D, Iigrave-cy.

I'm wondering why this duplication exists.

Thanks,
George Thomas

**Comment by tphinney** [comment]
Sounds like a bug! I'll look into it.

On Tue, Nov 17, 2015 at 8:03 PM, George Thomas <notifications@github.com>
wrote:

> Thomas:
> One of the unencoded entries in AC-3-exp.enc actually has a Unicode value.
> uni0438_uni0300 == uni045D, iigrave-cy. The only noticeable difference is
> the naming. uni0438_uni0300 translates as ii_gravecomb-cy using OS X
> Services. The same situation exists for the cap version, uni0418_uni0300 ==
> uni040D, Iigrave-cy.
>
> I'm wondering why this duplication exists.
>
> Thanks,
> George Thomas
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tphinney/font-tools/issues/4>.
>



-- 
“If friendship is magic, then any sufficiently advanced technology is
indistinguishable from friendship.”
—Shanna Germain (after Arthur C. Clarke)

**Comment by tphinney** [comment]
Yes, this is a bug in the source data I got from Adobe. I've alerted them
to the bug, and will adjust my files appropriately.

(Although, as they use combining marks for non-Unicode combinations, it's
not clear the bug would really hurt them any.)

On Wed, Nov 18, 2015 at 8:15 AM, Thomas Phinney <tphinney@cal.berkeley.edu>
wrote:

> Sounds like a bug! I'll look into it.
>
> On Tue, Nov 17, 2015 at 8:03 PM, George Thomas <notifications@github.com>
> wrote:
>
>> Thomas:
>> One of the unencoded entries in AC-3-exp.enc actually has a Unicode
>> value. uni0438_uni0300 == uni045D, iigrave-cy. The only noticeable
>> difference is the naming. uni0438_uni0300 translates as ii_gravecomb-cy
>> using OS X Services. The same situation exists for the cap version,
>> uni0418_uni0300 == uni040D, Iigrave-cy.
>>
>> I'm wondering why this duplication exists.
>>
>> Thanks,
>> George Thomas
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/tphinney/font-tools/issues/4>.
>>
>
>
>
> --
> “If friendship is magic, then any sufficiently advanced technology is
> indistinguishable from friendship.”
> —Shanna Germain (after Arthur C. Clarke)
>



-- 
“If friendship is magic, then any sufficiently advanced technology is
indistinguishable from friendship.”
—Shanna Germain (after Arthur C. Clarke)

**Comment by MajusCorp** [comment]
Thomas, I found one more in AC-3-exp.enc.

uni0415_uni0300, Ie_gravecomb-cy == Iegrave-cy, uni0400 and the 
lowercase, uni0435_uni0300, ie_gravecomb-cy == iegrave-cy, uni0450.

It is kind of odd, but as you said, it probably wouldn't cause any problems.

I also spot-checked the Latin 5 unencoded but found nothing.

Best,
George Thomas

**Comment by tphinney** [comment]
Thanks!

I've updated the file accordingly.

On Wed, Nov 18, 2015 at 3:42 PM, George Thomas <notifications@github.com>
wrote:

> Thomas, I found one more in AC-3-exp.enc.
>
> uni0415_uni0300, Ie_gravecomb-cy == Iegrave-cy, uni0400 and the
> lowercase, uni0435_uni0300, ie_gravecomb-cy == iegrave-cy, uni0450.
>
> It is kind of odd, but as you said, it probably wouldn't cause any
> problems.
>
> I also spot-checked the Latin 5 unencoded but found nothing.
>
> Best,
> George Thomas
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tphinney/font-tools/issues/4#issuecomment-157900100>.
>



-- 
“If friendship is magic, then any sufficiently advanced technology is
indistinguishable from friendship.”
—Shanna Germain (after Arthur C. Clarke)
